### DISCUSSION TITLE
does this work for magic link authentication

### DISCUSSION THREAD

**Discussion by nadiTime** [danestves/remix-auth-auth0#84] [General]
I get this log when authenticating via magic link

```
http://localhost:3902/auth/auth0/callback#access_token=[access_token]&scope=openid&expires_in=7200&token_type=Bearer

{
  "message": "Missing state on URL."
}
```
I am using the auth0 authentication strategy

**Comment by danestves** [comment]
Hello! Yeah, I got that error sometimes, currently investigating why is the issue

**Reply by nadiTime** [reply]
that's great thanks
seems like there are different url params for the magic link redirect....

**Reply by nadiTime** [reply]
you already get access token in the redirect
so i guess you should add this to the strategy?

**Reply by danestves** [reply]
Yeah! I'm a little blocked right now, but if you can help with a PR I will review it and merge it as soon as possible!
